rsocket/statemachine/RSocketStateMachine.cpp

```diff
@@ -77,6 +77,7 @@ void RSocketStateMachine::setResumable(bool resumable) {
 void RSocketStateMachine::connectServer(
     yarpl::Reference<FrameTransport> frameTransport,
     const SetupParameters& setupParams) {
+  stats_->socketConnected();
```

Comment:
It might be better to put this inside connect() function call.  So it will cover all cases -  client/server and reconnects/resumes